src/instance.js

```diff
@@ -0,0 +1,12 @@
+export const AlpineMagicHelpers = {
```

Comment:
Fundamentally, shouldn't we read window.AlpineMagicHelpers here and merge with defaults instead of setting the window object in each of the helpers?

Reply:
Yeah. I just noticed `AlpineMagicHelpers` is already exposed from the bundler too